### DISCUSSION TITLE
Dark mode on Qt5.15.1 darker, higher contrast

### DISCUSSION THREAD

**Discussion by deurzen** [qutebrowser/qutebrowser#5795] [Q&A]
Is it me, or has dark mode gotten significantly worse since Qt5.15.1 (up from Qt5.14)? It seems as though, under default settings, the contrast is higher, and the colors are darker overall. Altered web pages look a lot less "natural," I'd say.

**Comment by The-Compiler** [comment]
I converted this to a discussion - nothing qutebrowser can do here, the dark mode implementation is coming from the underlying Chromium.
